spec/08-pattern-matching.md

```diff
@@ -191,8 +191,8 @@ member method named `unapply` or `unapplySeq` that matches
 the pattern.
 
 An `unapply` method in an object $x$ _matches_ the pattern
-$x(p_1 , \ldots , p_n)$ if it takes exactly one argument and one of
```

Comment:
Sleeping on it, I guess this could also mean "can be called with a single argument". In which case we'd need to support `def unapply(xs: Foo*): Option[Bar]`, too...